starknet_py/tests/e2e/client/prepare_net_for_gateway_test.py

```diff
@@ -0,0 +1,50 @@
+from starknet_py.contract import Contract
+from starknet_py.net import AccountClient
+from starknet_py.transactions.declare import make_declare_tx
+
+
+def prepare_net_for_tests(account_client: AccountClient, compiled_contract: str):
```

Comment:
If #356 is merged, this could probably be changed to async function so don't have to use `_sync` methods.